### ISSUE TITLE
Suggest rewording summary for interface groups for clarity

### ISSUE THREAD

**Issue by wnichola-ntap** [NetAppDocs/ontap#1007]
### Page URL

https://docs.netapp.com/us-en/ontap/networking/combine_physical_ports_to_create_interface_groups.html

### Page title

Combine physical ports to create interface groups

### Summary

I think it would be more clear if we reworded the summary for interface groups to more explicitly declare that ONTAP does not support mixing ports from different cluster nodes into a single ifgrp. It's kind of present when you get to the actual example, but I think we could come right out and say it in the summary.

Suggest rewording this:

An interface group, also known as a Link Aggregation Group (LAG), is created by combining two or more physical ports into a single logical port. The logical port provides increased resiliency, increased availability, and load sharing.

to this:

An interface group, also known as a Link Aggregation Group (LAG), is created by combining two or more physical ports on the same node into a single logical port. The logical port provides increased resiliency, increased availability, and load sharing.

### Public issues must not contain sensitive information

- [X] This issue contains no sensitive information.

**Comment by netapp-barbe** [comment]
Reviewing update with engineering

**Comment by netapp-barbe** [comment]
Decision is to make the statement generic to cover any applications or features that are not bound by specific service policies. The statement is: "Features that are not bound to a specific service will use a system-defined behavior to select LIFs for outbound connections.

**Comment by netapp-barbe** [comment]
Documentation update complete: https://docs.netapp.com/us-en/ontap/networking/combine_physical_ports_to_create_interface_groups.html
